index.html

```diff
@@ -1,19 +1,106 @@
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="karma-logo" class="navbar-logo" />
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+      <nav>
+        <ul class="navbar-list">
+          <li id="navbar-button"><a href="#">Meet Karma</a></li>
+          <li id="navbar-button"><a href="#">How It Works</a></li>
+          <li id="navbar-button"><a href="#">Store</a></li>
+          <li id="navbar-button"><a href="#">Blog</a></li>
+          <li id="navbar-button"><a href="#">Help</a></li>
+          <li id="navbar-button"><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!--main starts here-->
+    <main>
+      <section class="main-articles">
+        <article class="article-one">
+          <img
+            src="img/first-background.jpg"
+            alt="first-background"
+            class="background-image"
+          />
+          <h1 class="line-one">Introducing Karma</h1>
+          <h2 class="line-two">Bring WiFi with you, everywhere you go.</h2>
+          <button class="button">Learn More</button>
+        </article>
 
-</body>
+        <article class="article-two">
+          <h3 class="little-karma">Everyone Needs a Little Karma.</h3>
+          <ul class="icones-list">
+            <li class="photo-text">
+              <img
+                src="img/icon-devices.svg"
+                alt="devices-icon"
+                class="devices"
+              />
+              <a href="#">Internet for all devices</a>
+            </li>
+            <li class="photo-text">
+              <img src="img/icon-coffee.svg" alt="coffee-icon" class="coffee" />
+              <a href="#">Boost your productivity </a>
+            </li>
+            <li class="photo-text">
+              <img src="img/icon-refill.svg" alt="refill-icon" class="refill" />
+              <a href="#">Pay as You Go</a>
+            </li>
+          </ul>
+        </article>
+      </section>
+    </main>
+    <!--footer starts here-->
+    <hr />
+    <footer class="footer">
+      <h4 class="join-us">Join us on</h4>
+      <ul class="socialmedia-list">
+        <li class="icons">
+          <a href="#">
+            <img
+              src="img/twitter-icon.svg"
+              alt="twitter-icon"
+              class="twitter-icon"
+          /></a>
+        </li>
+        <li class="icons">
+          <a href="#">
+            <img
+              src="img/facebook-icon.svg"
+              alt="facebook-icon"
+              class="facebook-icon"
+          /></a>
+        </li>
+        <li class="icons">
+          <a href="#">
+            <img
+              src="img/instagram-icon.svg"
+              alt="instagram-icon"
+              class="instagram-icon"
+          /></a>
+        </li>
+      </ul>
+      <h6 class="mobility">@ Karma Mobility, Inc.</h6>
```

Comment:
This doesn't need to be a heading as there's no content after it. `<p />` or `<div />` or `<small />` is fine